source/theory.rst

```diff
@@ -227,6 +241,10 @@ where :math:`H` is the wave height, :math:`\omega` is the wave frequency  (:math
 
 Wave Spectra
 ^^^^^^^^^^^^^^^^^^^^^^^
+.. Note:: 
+	Adam:
+	Use different coefficients for A, B, C or add subscripts to avoid confusion with the BEM and PTO coefficients
+
```

Comment:
I agree but we need to do this carefully. Those are industry standard variables, so i would recommend keeping them but adding subscripts to differentiate.

Reply:
@kmruehl Good point. I updated them all with subscripts to prevent confusion, and added each instance to the terminology so the differences are clear.